control-plane-operator/controllers/hostedcontrolplane/scheduler/config_test.go

```
@@ -0,0 +1,51 @@
+package scheduler
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/openshift/hypershift/support/config"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	componentbasev1 "k8s.io/component-base/config/v1alpha1"
+	schedulerv1beta2 "k8s.io/kube-scheduler/config/v1beta2"
+	"k8s.io/utils/pointer"
+)
+
+func TestGenerateConfig(t *testing.T) {
+	leaseDuration, _ := time.ParseDuration(config.RecommendedLeaseDuration)
```

instead of ignoring the error, foreach:

```
g := NewWithT(t)
g.Expect(err).ShouldNot(HaveOccurred())
```